packages/tailwindcss/src/utilities.ts

```diff
@@ -1381,70 +1441,74 @@ export function createUtilities(theme: Theme) {
       return `${value}%`
     },
     handle: (value) => [
-      scaleProperties(),
+      ...scaleProperties(),
       decl('--tw-scale-x', value),
       decl('--tw-scale-y', value),
-      decl('scale', 'var(--tw-scale-x) var(--tw-scale-y)'),
     ],
   })
 
-  /**
-   * @css `scale`
-   */
-  functionalUtility('scale-x', {
-    supportsNegative: true,
-    themeKeys: ['--scale'],
-    handleBareValue: ({ value }) => {
-      if (Number.isNaN(Number(value))) return null
-      return `${value}%`
+  suggest('scale', () => [
+    {
+      supportsNegative: true,
+      values: ['0', '50', '75', '90', '95', '100', '105', '110', '125', '150', '200'],
+      valueThemeKeys: ['--scale'],
     },
-    handle: (value) => [
-      scaleProperties(),
-      decl('--tw-scale-x', value),
-      decl('scale', 'var(--tw-scale-x) var(--tw-scale-y)'),
-    ],
-  })
+  ])
+
+  for (let axis of ['x', 'y', 'z']) {
+    /**
+     * @css `scale`
+     */
+    functionalUtility(`scale-${axis}`, {
+      supportsNegative: true,
+      themeKeys: ['--scale'],
+      handleBareValue: ({ value }) => {
+        if (Number.isNaN(Number(value))) return null
+        return `${value}%`
+      },
+      handle: (value) => [...scaleProperties(), decl(`--tw-scale-${axis}`, value)],
+    })
+
+    suggest(`scale-${axis}`, () => [
+      {
+        supportsNegative: true,
+        values: ['0', '50', '75', '90', '95', '100', '105', '110', '125', '150', '200'],
+        valueThemeKeys: ['--scale'],
+      },
+    ])
+  }
 
   /**
    * @css `scale`
    */
-  functionalUtility('scale-y', {
+  functionalUtility('scale-3d', {
     supportsNegative: true,
     themeKeys: ['--scale'],
-    handleBareValue: ({ value }) => {
-      if (Number.isNaN(Number(value))) return null
-      return `${value}%`
-    },
+    handleBareValue: ({ value }) => `${value}%`,
```

Comment:
I think something went wrong while rebasing, this should have some validation:
```suggestion
    handleBareValue: ({ value }) => {
       if (Number.isNaN(Number(value))) return null
       return `${value}%`
     },
```

Otherwise, we can type `scale-3d-potato%`